frontend-react/package.json

```diff
@@ -71,7 +71,7 @@
         "cypress:production:open": "echo 'not implemented yet'",
         "test": "jest",
         "test:debug": "jest --watch --runInBand",
-        "test:ci": "jest --ci --runInBand --coverage --verbose --silent",
```

Comment:
Why would we want to silence the output from the github action logs?

Reply:
ah yeah, whoops. that was a vestige of experimentation. will remove (also `--verbose` + `--silent` == comedy)

Reply:
Oh! I removed the `--silent` for that reason! Also `--silent` + `--verbose` doesn't make any sense together.

Reply:
Woops, went dyslexic on reading the diff line here. ignore